src/plonemeeting/portal/core/tests/test_filters.py

```
@@ -1,81 +1,102 @@
-                        u'<p>Au scrutin secret et à l’unanimité, de désigner <span class="pm-anonymize"></span>, ' \
-                        u'en qualité d’instituteur maternel temporaire mi-temps en remplacement ' \
-                        u'de <span class="pm-anonymize"></span> aux écoles communales fondamentales.</p>'
+        self.richtext = (
+            "<p><strong>Article 1er</strong> :</p>"
+            '<p>Au scrutin secret et à l’unanimité, de désigner <span class="pm-anonymize"></span>, '
+            "en qualité d’instituteur maternel temporaire mi-temps en remplacement "
+            'de <span class="pm-anonymize"></span> aux écoles communales fondamentales.</p>'
+        )
 
     def test_replace_masked_gdpr(self):
         filter = ReplaceMaskedGDPR(self.institution, None)
```

Ruff (0.15.14)

[error] Variable `filter` is shadowing a Python builtin

(A001)